### PR TITLE
Add few information to memory assert message.

### PR DESCRIPTION
**Pull request by yueawang**
## What changes were proposed in this pull request?

Add few information to assert message to help debugging.

## How was this patch tested?
mvn passes.

### CODE REVIEW

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/MemoryManager.scala [hunk 1]
```diff
@@ -171,6 +171,7 @@ private[oap] object MemoryManager extends Logging {
     assert(SparkEnv.get != null, "Oap can't run without SparkContext")
     val memoryManager = SparkEnv.get.memoryManager
     // TODO: make 0.7 configurable
+    assert(memoryManager.maxOffHeapStorageMemory > 0, "Oap can't run without offHeap memory")
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/MemoryManager.scala]: Thanks. this is what I always forgot.

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala [hunk 2]
```diff
@@ -124,7 +124,8 @@ class GuavaOapCache(cacheMemory: Long, cacheGuardianMemory: Long) extends OapCac
   override def get(fiber: Fiber, conf: Configuration): FiberCache = {
     val fiberCache = cache.get(fiber, cacheLoader(fiber, conf))
     // Avoid loading a fiber larger than MAX_WEIGHT / 4, 4 is concurrency number
-    assert(fiberCache.size() <= MAX_WEIGHT * MB / 4, "Can't cache fiber larger than MAX_WEIGHT / 4")
+    assert(fiberCache.size() <= MAX_WEIGHT * MB / 4,
+      s"Can't cache fiber(${fiberCache.size()}) larger than MAX_WEIGHT($MAX_WEIGHT) / 4")
```
Comment (linhong-intel) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: Pleas use the Utils.bytesToString to format size.

Comment (linhong-intel) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: Also, please hep one of my mistake.
I remove the concurrencyLevel already, so we need to remove `4` as well.

Comment (yueawang) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: I remember guava has a default concurrency number 4?

Comment (linhong-intel) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: you are right..... my bad

Comment (carsonwang) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: Not related to this PR. I just have a little concern on the hard coded 4 here. If guava changes the default concurrency in the next version and we update to that version, it is very likely we will forget to update this number here. If there is no variable in guava that represents the default concurrency, we may still want to specify the concurrency ourselves and use the same variable here. @linhong-intel

**Comment by adrian-wang** [comment]
merged to master and branch-0.3

**Pull request merged**